Preserve framebuffer contents on initialization.

This avoids a black flash while transitioning from the static boot image and
boot animation.

---
 libgralloc/a-family/framebuffer.cpp |    1 -
 libgralloc/badger/framebuffer.cpp   |    1 -
 2 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/libgralloc/a-family/framebuffer.cpp b/libgralloc/a-family/framebuffer.cpp
index e107312..b7c4379 100755
--- a/libgralloc/a-family/framebuffer.cpp
+++ b/libgralloc/a-family/framebuffer.cpp
@@ -956,7 +956,6 @@ int mapFrameBufferLocked(struct private_module_t* module)
         return -errno;
     }
     module->framebuffer->base = intptr_t(vaddr);
-    memset(vaddr, 0, fbSize);
 
 #if defined(HDMI_DUAL_DISPLAY)
     /* Overlay for HDMI*/
diff --git a/libgralloc/badger/framebuffer.cpp b/libgralloc/badger/framebuffer.cpp
index 63587c6..6554ac2 100644
--- a/libgralloc/badger/framebuffer.cpp
+++ b/libgralloc/badger/framebuffer.cpp
@@ -988,7 +988,6 @@ int mapFrameBufferLocked(struct private_module_t* module)
         return -errno;
     }
     module->framebuffer->base = intptr_t(vaddr);
-    memset(vaddr, 0, fbSize);
 
 #if defined(HDMI_DUAL_DISPLAY)
     /* Overlay for HDMI*/
-- 
1.7.0.4

